src/components/contact/Contact.js

```diff
@@ -24,14 +22,17 @@ const Contact = ({ t }) => {
         <img src={peopleSearch} alt="people search logo" />
       </div>
       <div className={styles.rightSection}>
-        <h1>{t('contact-page-header')}</h1>
+        <h1>
+          <Trans id="contact-page-header" />
+        </h1>
         <p>
-          <Interpolate i18nKey="contact-page-msg" slackOrg={slackOrg} email={emailLink} />
+          We want to hear your thoughts! Feel free to join our {slackOrg} or send us an email at{' '}
```

Comment:
Let's use an id for this: `<Trans id="contact.msg">We want to hear your thoughts! Feel free to join our {slackOrg} or send us an email at</Trans>`